gcp-jobs/involuntary-dissolutions/src/involuntary_dissolutions/config.py

```diff
@@ -0,0 +1,114 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""All of the configuration for the service is captured here."""
+
+import os
+
+from dotenv import find_dotenv, load_dotenv
+
+load_dotenv(find_dotenv())
+
+CONFIGURATION = {
+    "development": ".config.DevConfig",
+    "testing": ".config.TestConfig",
+    "production": ".config.ProdConfig",
+    "default": ".config.ProdConfig"
+}
+
+
+def get_named_config(config_name: str = "production"):
+    """Return the configuration object based on the name."""
+    if config_name in ["production", "staging", "default"]:
+        config = ProdConfig()
+    elif config_name == "testing":
+        config = TestConfig()
+    elif config_name == "development":
+        config = DevConfig()
+    else:
+        raise KeyError(f"Unknown configuration '{config_name}'")
+    return config
+
+
+class _Config:  # pylint: disable=too-few-public-methods
+    """Base class configuration."""
+
+    SERVICE_NAME = "dissolutions-job"
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    LD_SDK_KEY = os.getenv("LD_SDK_KEY", None)
+
+    SERVERS = os.getenv("SERVERS", None)
+    CLUSTER_ID = os.getenv("CLUSTER_ID", None)
+    CLIENT_NAME = os.getenv("CLIENT_NAME", None)
+    FILER_SUBJECT = os.getenv("FILER_SUBJECT", "entity.filer")
```

Comment:
this should be set to env var 'BUSINESS_FILER_TOPIC'. To avoid confusion I think it makes sense to rename the config variable to FILER_TOPIC

Reply:
Changed